src/Symfony/Component/HttpFoundation/File/File.php

```diff
@@ -104,6 +104,11 @@ public function move(string $directory, string $name = null)
         return $target;
     }
 
+    public function getContent(): string
```

Comment:
getContents?

Reply:
HA
I'm always wondering about this.
I Symfony, we always use `getContent()`... (HttpFoundation, HttpCLient, BrowserKit, etc.)
In other places, there is `getContents()` (SplFileInfo, PSR-7, etc.)

Reply:
ahah, thought the same. Indeed in Symfony we use `getContent()`. I prefered to be consistant